packages/crew_integration.rb

```diff
@@ -0,0 +1,78 @@
+require 'package'
+
+class Crew_integration < Package
+  description 'Improves integration with Chrome OS shell and Chromebrew GUI applications.'
+  homepage 'https://github.com/skycocker/chromebrew/crew_integration'
+  version '2.0'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on "graphicsmagick"
+    
+  case ARGV[0]
+  when 'reinstall', 'install'
+     FileUtils.rm_rf "#{CREW_PREFIX}/lib/pwa/"
+     FileUtils.rm_rf "#{HOME}/MyFiles/.extension"
+  end
+    
+  case ARGV[0]
+  when 'reinstall'
+     puts "Since you are reinstalling this package, please run 'pkill ruby' if installation failed".lightblue
+  when 'upgrade'
+     puts "Since you are upgrading this package, please run 'pkill ruby' if installation failed".lightblue
+  end
```

Comment:
Why not just run `pkill ruby` as part of the process?

Reply:
Isn't `crew` based on ruby lol

Reply:
Not as a service though, right?  I was thinking something like:
```
  case ARGV[0]
  when 'reinstall', 'install'
     system 'pkill ruby'
     FileUtils.rm_rf "#{CREW_PREFIX}/lib/pwa/"
     FileUtils.rm_rf "#{HOME}/MyFiles/.extension"
  end
```

Reply:
The installation will stop immediately after `pkill`, I am finding a better solution on closing the websocket connection now:)